waku/factory/external_config.nim

```diff
@@ -589,6 +589,22 @@ type WakuNodeConf* = object
       name: "websocket-secure-cert-path"
     .}: string
 
+    ## Rate limitation config
+    ## Currently default to switch of rate limit until become official
+    requestRateLimit* {.
+      desc:
+        "Number of requests to serve by each service in the specified period. Set it to 0 for unlimited",
+      defaultValue: 0,
+      name: "request-rate-limit"
+    .}: int
+
+    ## Currently default to switch of rate limit until become official
+    requestRatePeriod* {.
+      desc: "Period of request rate limitation in seconds. Set it to 0 for unlimited",
+      defaultValue: 0,
+      name: "request-rate-period"
+    .}: int64
```

Comment:
Maybe you already commented and I might be missing something but these params seem not to be used :)